docs/zh/command/creport.md

```
@@ -0,0 +1,361 @@
+
+**creport 主要用于查询集群指定时间范围的用户、账户相关作业统计信息。**
+
+```bash
+creport [<OPTION>] [<COMMAND>]
+```
+
+## 展示用户资源消耗排行
+```bash
+creport user topusage [--start-time=...] [--end-time=...] [--account=...] ...
+```
+**输出示例**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)